### ISSUE TITLE
Optimizer ADAM used in Learning ADAMW in traversability estimator

### ISSUE THREAD

**Issue by JonasFrey96** [leggedrobotics/wild_visual_navigation#194]
https://github.com/leggedrobotics/wild_visual_navigation/blob/fa94554344b305e50711166fcc4af3175685a684/wild_visual_navigation/learning/lightning/lightning_module.py#L436

Currently I ablate the learning rate and the optimizer I think it is smartest to only consider ADAM and SGD. We do a hyperparameter sweep and use whatever works best.
